lib/tests/streamlit/runtime/caching/storage/local_disk_cache_storage_test.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for LocalDiskCacheStorage and LocalDiskCacheStorageManager"""
+import logging
+import math
+import os.path
+import shutil
+import unittest
+from unittest.mock import MagicMock, patch
+
+from testfixtures import TempDirectory
+
+from streamlit import util
+from streamlit.logger import get_logger
+from streamlit.runtime.caching.storage import (
+    CacheStorageContext,
+    CacheStorageError,
+    CacheStorageKeyNotFoundError,
+)
+from streamlit.runtime.caching.storage.in_memory_cache_storage_wrapper import (
+    InMemoryCacheStorageWrapper,
+)
+from streamlit.runtime.caching.storage.local_disk_cache_storage import (
+    InMemoryWrappedLocalDiskCacheStorageManager,
+    LocalDiskCacheStorage,
+)
+
+
+class InMemoryWrappedLocalDiskCacheStorageManagerTest(unittest.TestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.tempdir = TempDirectory(create=True)
+        self.patch_get_cache_folder_path = patch(
+            "streamlit.runtime.caching.storage.local_disk_cache_storage.get_cache_folder_path",
+            return_value=self.tempdir.path,
+        )
+        self.patch_get_cache_folder_path.start()
+
+    def tearDown(self) -> None:
+        super().tearDown()
+        self.patch_get_cache_folder_path.stop()
+        self.tempdir.cleanup()
+
+    def test_create_persist_context(self):
```

Comment:
Can you add short docstrings to this and the rest of the tests in the file explaining what's being tested?